geospatial_map/v1/geospatial_map.yaml

```diff
@@ -0,0 +1,424 @@
+
+            - `Unsupported`: the USS cannot process the dataset type specified.
+
+            - `Rejected`: the geospatial data source was rejected because it is invalid.
+
+            - `Error`: the geospatial data activation or deactivation failed. The message field is required in this case.
+          type: string
+          enum: [Activating, Ready, Deactivating, Deactivated, Unsupported, Rejected, Error]
+          example: Ready
+        message:
+          description: Human-readable explanation of the result for debugging purpose only. This field is required when the result value is `Error`.
+          type: string
+          example: |-
+            Unable to download the dataset https://caa.example.com/geozones.json. Connection refused.
+
+    Position:
+      type: object
+      required:
+        - uomDimensions
+        - verticalReferenceType
+      properties:
+        uomDimensions:
+          $ref: './ed269.yaml#/components/schemas/UomDimensions'
+        verticalReferenceType:
+          $ref: './ed269.yaml#/components/schemas/VerticalReferenceType'
+        height:
+          description: >-
+            Height above vertical reference datum indicated in `verticalReferenceType`, in units of `uomDimensions`.
+          type: number
+          default: 0
+        longitude:
+          description: >-
+            Longitude, degrees east of prime meridian.
+          type: number
+          default: 0
+        latitude:
+          description: >-
+            Latitude, degrees north of the equator.
+          type: number
+          default: 0
+
+    GeospatialMapQueryRequest:
+      type: object
+      required:
+        - checks
+      properties:
+        checks:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeospatialMapCheck'
+
+    GeospatialMapCheck:
+      type: object
+      required:
+        - filterSets
+      properties:
+        filterSets:
+          description: >-
+            Select geospatial features which match any of the specified filter sets.
+          type: array
+          items:
+            $ref: '#/components/schemas/GeospatialFeatureFilterSet'
+
+    GeospatialFeatureFilterSet:
+      description: >-
+        Set of filters to select only a subset of geospatial features.  Only geospatial features which are applicable to all specified filters within this filter set should be selected.
+      type: object
+      properties:
+        position:
+          description: >-
+            If specified, only select geospatial features encompassing this position.
+          anyOf:
+            - $ref: '#/components/schemas/Position'
+        volumes4d:
+          description: >-
+            If specified, only select geospatial features at least partially intersecting one or more of these volumes.
+          type:
+            array
+          items:
+            $ref: './astm_f3548_21.yaml#/components/schemas/Volume4D'
```

Comment:
Yeah...this one I wasn't sure how best to handle.  This PR currently adds SFC as an allowed enum value in the Altitude of astm_f3548_21.yaml, which I think is ok because the data structures in that file merely "align" with F3548-21 rather than "reflect", "duplicate", or something similar.  I think it's fair that the volume data structures here should be independent of F3548-21 even if they align, so maybe labeling them as relating to F3548-21 isn't optimal.  But, I don't think duplicating 180 lines without the context that it's mostly the same stuff as F3548-21 is optimal either.

Good point on alignment with Position; I've updated that one.  That does use less of the ED-269, but I think within-API harmonization is probably more important.

Reply:
Added ft to units